base/shell/explorer/startmnucust.cpp

```diff
@@ -66,40 +66,213 @@ VOID OnClearRecentItems()
+    {
+        hitem.iItem = i;
+        LoadStringW(NULL, id++, chBuffer, 60);
+        hitem.pszText = chBuffer;
+        hitem.cchTextMax = 60;
+        SendMessage(hListView, LVM_INSERTITEM, 0, (LPARAM)&hitem);
+		
+        DWORD dwSize = 32;
+        DWORD dwType;
+		
+        if (RegQueryValueExW(hKey, ClassicOptStr[i], NULL, &dwType, (LPBYTE) chBuffer, &dwSize) == ERROR_SUCCESS )
+        {			
+            if (dwType == REG_SZ)
+            {
+                BOOL stat = lstrcmp(chBuffer,L"YES") ? FALSE : TRUE;
+                ListView_SetCheckState(hListView, i, stat);				
+            }
+            else if (dwType == REG_DWORD)
+            {
+                DWORD regValue = *(reinterpret_cast<DWORD*>(chBuffer));
+                ListView_SetCheckState(hListView, i, regValue);
+            } 		  
+        }
+    }
+    RegCloseKey(hKey); 
+    return hListView;
+}
+
+void UpdateClassicOptions(HWND hListView)
+{
+    int nItemCount = ListView_GetItemCount(hListView);
+    HKEY hKey;
+    if (RegOpenKeyExW(HKEY_CURRENT_USER,
+                      L"Software\\Microsoft\\Windows\\Currentversion\\Explorer\\Advanced",
+                      0,
+                      KEY_SET_VALUE,
+                      &hKey) != ERROR_SUCCESS)
+        return;
+    for(int nItem = 0; nItem < nItemCount; nItem++)
+    {
+        if (ClassicOptDirty[nItem] == TRUE)
+        {
+            BOOL state = ListView_GetCheckState(hListView, nItem);
+            switch(nItem)
+            {
+                case 2:
+                case 3:
+                case 4:
+                case 5:
+                case 6:
+                case 7:
+                case 9:
+                {
+                    const WCHAR *sKey = state ? L"YES" : L"NO";
+                    RegSetValueExW(hKey, ClassicOptStr[nItem], NULL, REG_SZ, (LPBYTE)sKey, (lstrlen(sKey)+1)*sizeof(WCHAR));
+                    break;
+                }
+                default:
+                {
+                    DWORD regValue = state ? 1 : 0;
+                    DWORD dwSize = 4;
+                    RegSetValueExW(hKey, ClassicOptStr[nItem], NULL, REG_DWORD, (LPBYTE)&regValue, dwSize);
+                    break;
+                }
+            }
+            ClassicOptDirty[nItem] = FALSE;
+        }
+    } 
+    RegCloseKey(hKey);		
+}
+
+INT_PTR CALLBACK CustomizeClassicProc(HWND hwnd, UINT Message, WPARAM wParam, LPARAM lParam)
+{
+    static HWND hListView = NULL;
+	
+    switch(Message)
+    {
+        case WM_INITDIALOG:
+             /* FIXME: Properly initialize the dialog (check whether 'clear' button must be disabled, for example) */
+            hListView = InitClassicListView(hwnd);          
+            return TRUE;
+        case WM_NOTIFY:
+            if (IDC_CLASSICSTART_SETTINGS == ((LPNMHDR)lParam)->idFrom && ((LPNMHDR)lParam)->code == NM_CLICK)
+            {
+                int item = ((LPNMITEMACTIVATE)lParam)->iItem;
+                ClassicOptDirty[item] = TRUE;				
```

Comment:
Trailing whitespace again....... here...

Reply:
Finally solved the white space problem.
I could not see them! It was like looking for ghosts.
I can see them now.

Reply:
You need to use some editor like Notepad++ and enable the visualization of whitespace and tabs in "View"/"Show symbols" menu.